### DISCUSSION TITLE
Unintensionally deleted previous account.

### DISCUSSION THREAD

**Discussion by meetalif** [community/community#53587] [New to GitHub] [answered]
Hello GitHub Support,

I am writing to request your help with an issue regarding my account and username. My previous account, meetalif, was deleted by my cousin who wanted to delete a repository, but he accidentally deleted the whole account. I have created a new account with the same email, but I am unable to use meetalif as my username. This is a problem for me because I use meetalif as my handle for all my online platforms and it is important for my business.

Is there any way you can restore my previous account or free up the username meetalif so I can use it for my new account? I would really appreciate your assistance with this matter as soon as possible.

Thank you for your time and attention.

Sincerely, Alif

**Comment by queenofcorgis** [comment, marked as answer]
Hi @meetalif0 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). The Community is unable to assist in matters like these so I'll be locking the post. Please open a ticket with them :)
